layouts/error.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<div>
+		<div class="error-container">
+			<template v-if="error.statusCode === 404">
+				<h1>Página não encontrada!</h1>
+
+				<img :src="require('~/static/illustrations/searching.png')" />
+			</template>
+
+			<template v-else>
+				<h1>Ocorreu algo inesperado!</h1>
+
+				<img :src="require('~/static/illustrations/bored.png')" />
+			</template>
+
+			<NuxtLink to="/">
+				<button type="button" class="back-button">Voltar</button>
+			</NuxtLink>
+		</div>
+	</div>
+</template>
+
+<script>
+	export default {
+		layout: 'default',
+		props: {
+			error: {
+				type: Object,
+				default: null
+			}
+		},
+
+		methods: {
+			backToHome() {
```

Comment:
O ideal seria esse método se chamar "sendBackToHome()", mas como ele não ta sendo utilizado ainda tu pode escolher renomear agora ou na próxima vez que for mexer no layout, só tem que lembrar de alterar daí kkkkk